data/united-states/south-dakota.csv

```diff
@@ -1,27 +1,46 @@
 monks-ale-house-sioux-falls,Monk's Ale House,brewpub,420 E 8th St,,,Sioux Falls,South Dakota,,57103-7025,http://www.monkshouseofalerepute.com,6053382328,7/24/2018,10/1/2021,United States,-96.7218594,43.5488749,
-mt-rushmore-brewing-company-custer,Mt. Rushmore Brewing Company,brewpub,140 Mt Rushmore Rd,,,Custer,South Dakota,,57730-1829,http://www.mtrushmorebrewing.com,,7/24/2018,8/24/2018,United States,-103.605392,43.764722,
 one-legged-pheasant-brewery-aberdeen,One Legged Pheasant Brewery,micro,721 S Main St,,,Aberdeen,South Dakota,,57401-6047,http://oneleggedpheasant.com,6057251534,7/24/2018,10/1/2021,United States,-98.48790197,45.45741525,
 panacea-brewing-company-remedy-brewing-company-sioux-falls,Panacea Brewing Company / Remedy Brewing Company,brewpub,401 E 8th St Ste 120,,,Sioux Falls,South Dakota,,57103-7012,http://www.remedybrewco.com,6052141327,7/24/2018,10/1/2021,United States,-96.7221603,43.54794988,
 severance-brewing-company-sioux-falls,Severance Brewing Company,micro,701 N Phillips Ave,,,Sioux Falls,South Dakota,,57104-1623,http://severancebeer.com/,6052715480,7/24/2018,10/1/2021,United States,-96.72724272,43.5560742,
 sick-n-twisted-brewing-co-hill-city,Sick N Twisted Brewing Co,brewpub,23851 Highway 385 Ste A,,,Hill City,South Dakota,,57745-6544,http://www.sickntwistedbrewery.com,6055749200,7/24/2018,10/1/2021,United States,-103.5272662,43.94601735,
 spearfish-brewing-company-spearfish,Spearfish Brewing Company,micro,741 N Main St Ste 130,,,Spearfish,South Dakota,,57783-4147,http://spearfishbrewing.com,6057176999,7/24/2018,10/1/2021,United States,-103.8597028,44.4910696,
 the-knuckle-brewing-company-sturgis,The Knuckle Brewing Company,brewpub,918 Harley Davidson Way,,,Sturgis,South Dakota,,57785-8511,http://www.theknuckle.com,6055619846,7/24/2018,10/1/2021,United States,-103.5124614,44.41556787,
-wooden-legs-brewing-co-brookings,Wooden Legs Brewing Co,brewpub,309 5th St Ste 100,,,Brookings,South Dakota,,57006-1963,http://www.woodenlegsbrewing.com,6056922337,7/24/2018,8/24/2018,United States,-96.7996148,44.3104741,
 woodgrain-brewing-company-sioux-falls,WoodGrain Brewing Company,micro,101 S Phillips Ave Ste 100,,,Sioux Falls,South Dakota,,57104-6719,http://www.woodgrainbrew.com,6053105316,7/24/2018,10/1/2021,United States,-96.72722416,43.54728383,
+a-homestead-brew-valley-springs,A Homestead Brew,micro,26685 486th Ave,,,Valley Springs,South Dakota,,57068-7309,http://www.ahomesteadbrew.com,6055535015,10/1/2021,,United States,-96.49421923,43.51733014,
+armour-brewing-company-llc-armour,"Armour Brewing Company, LLC",brewpub,721 Main St,,,Armour,South Dakota,,57313-2122,http://www.facebook.com/ArmourBrewingCompany,4028901550,10/1/2021,,United States,-98.34450702,43.33084265,
+backspace-brewing-yankton,Backspace Brewing,micro,200 Walnut St,,,Yankton,South Dakota,,57078-4343,http://www.backspacebeer.com,4026508158,10/1/2021,,United States,-97.393642,42.86878841,
+,,,,,,,,,,,,,,,,,

```

Comment:
Looks like there were some extra rows added. Let's remove them to keep things tidy. Thanks!

Reply:
Will do

Reply:
Extra commas have been removed (I think, still new to PRs) :)